### PR TITLE
Archive proxyCoap work

### PR DESCRIPTION
**Pull request by fgalan**
Partial implementation of #1202

### CODE REVIEW

File: CHANGES_NEXT_RELEASE [hunk 1]
```diff
@@ -12,3 +12,4 @@ Fix:  Update PATCH request without payload must return error code 411 (#1257)
 Fix:  Clearer error 'entity not found' and 'conflict too many results' on NGSIv2 update PATCH (#1260)
 Add:  contextBroker standard error log in RPM init script (#1175)
 Fix:  avoid noisy error output in RPM init script (#309)
+Fix:  proxyCoap binary has been removed from RPM (#1202)
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE]: At the end, I didn't need to touch the .spec file to achieve this.

Comment (fgalan) [inline CHANGES_NEXT_RELEASE]: NTC

File: docker/Dockerfile [hunk 2]
```diff
@@ -3,7 +3,6 @@ FROM centos:centos6.6
 MAINTAINER FIWARE Orion Context Broker Team. Telefónica I+D
 
 ENV ORION_USER orion
-ENV GIT_REV_ORION_PROXYCOAP 749e22376664dd3adae17492090e58882d3b28a7
```
Comment (fgalan) [inline docker/Dockerfile]: @LeandroGuillen please check that docker building still works after the changes I have introduced (and that all the CAOP stuff has been removed and no leftover remains).

File: CHANGES_NEXT_RELEASE (hunk 1 above)
Comment (fgalan) [inline CHANGES_NEXT_RELEASE]: BTW, I have checked that an RPM generated at this branch doesn include the proxyCoap file:

```
[fermin@centollo fiware-orion]$ rpm -qlp /home/fermin/src/fiware-orion/rpm/RPMS/x86_64/contextBroker-0.24.0_next-dev.x86_64.rpm
/etc/cron.d/cron-logrotate-contextBroker-size
/etc/init.d/contextBroker
/etc/logrotate.d/logrotate-contextBroker-daily
/etc/prelink.conf.d/contextBroker.conf
/etc/sysconfig/contextBroker
/etc/sysconfig/logrotate-contextBroker-size
/usr/bin/contextBroker
/usr/share/contextBroker/garbage-collector.py
/usr/share/contextBroker/latest-updates.py
/usr/share/contextBroker/monit_log_processing.py
/usr/share/doc/contextBroker/LICENSE
```

File: docker/Dockerfile (hunk 2 above)
Comment (LeandroGuillen) [inline docker/Dockerfile]: It builds fine. There was a line leftover taken out., though.

**Comment by LeandroGuillen** [comment]
LGTM

**Pull request merged**